static/js/apps/diff/app.tsx

```diff
@@ -42,20 +42,35 @@ export function App(): JSX.Element {
   const [url, setUrl] = useState<AppUrl>(INIT_URL);
   const [inputUrl, setInputUrl] = useState<AppUrl>(INIT_URL);
 
+  useEffect(() => {
```

Comment:
I'm not sure the event passing is necessary here. If the goal is to keep the two iframes in sync, could you just set a local react state variable in this component from the iframe's onLoad and have the the relevant iframe re-render based on that state?

Reply:
I tried that but it's not possible due to cross origin check. Googled around and this seems to be a popular way of doing it.